src/index.js

```diff
@@ -1,93 +1,59 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
-
-const isProduction = process.env.NODE_ENV === "production";
-
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import errorhandler from 'errorhandler';
+import morgan from 'morgan';
 
 const app = express();
+const isProduction = process.env.NODE_ENV;
```

Comment:
Is there a .env file?

Reply:
@kipropbrian  this `production` environment is set in `start script` of `package.json` file